notebooks/Notebook1.md

```diff
@@ -1,34 +1,61 @@
 
 ## Description
 
-**TODO:** Fill in this part with information about your work this week:
-important design decisions, changes to previous decisions, open questions,
-exciting milestones, preliminary results, etc. Feel free to include images
-(e.g., a sketch of the design or a screenshot of a running program), links to
-code, and any other resources that you think will help clearly convey your
-design process.
+I began the week early last Friday because I was bored. I began by building a chart that would represent most if not all combinations of strings/inputs for my language. This helped me to map out how an input would be parsed into groups via regex pattern(see image below). Next I built a class with a regex pattern and a file parser. The file parser reads one line from a text file and uses the regex pattern on the input string/line to determine if it fits the pattern. If it does it parses the words from that line into the five groups mentioned earlier and below. 
+(playtype- a string or character uppercase or lowercase that must start out the string)
+(nameOrNumber - the name or number of a player, names must be a word where the first letter is capital and the rest lowercase, numbers can be any number 0-99)
+(numyards-this will be a number immediately followed by the letter y)
+(nameOrNumber2- same as previous)
+(outcome- a word in all caps representing the outcome of a play or an additional tag for a play)
+This aspect of my project was pretty straight forward and had little to no hiccups except for some experimentation with the regex pattern ([link to code here](https://github.com/hmc-cs111-spring2023/BBeckerArtifact/blob/main/src/Parser/StringToGroup.java)). Error handling for the parser has not yet been implemented but will be in the near future.
+chart image:
+
+<img width="602" alt="Screen Shot 2023-03-30 at 7 55 34 PM" src="https://user-images.githubusercontent.com/97985236/229011324-73a37a60-9a6c-4df1-becb-0fb27c6c819e.png">
+                                                                 
```

Comment:
I really like this technique for getting organized and planning the implementation!